static/monorail-2.0.yaml

```diff
@@ -2590,6 +2590,77 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+
+  /nodes/{identifier}/obm:
+    x-swagger-router-controller: nodes
+    get:
+      operationId: nodesGetObmsByNodeId
+      x-privileges: [ 'Read' ]
+      x-authentication-type: [ 'jwt' ]
+      x-view: obm.2.0.json
+      summary: |
+        Get an OBM service
```

Comment:
I think this summary need some change, this API is to get ALL obm service for the specified node.